test/e2e/service_bootstrap.py

```diff
@@ -42,7 +42,7 @@ def service_bootstrap() -> Resources:
             "ack-access-entry-principal-role",
             "eks.amazonaws.com",
         ),
-        ClusterVPC=VPC(name_prefix="cluster-vpc", num_public_subnet=2, num_private_subnet=2)
+        ClusterVPC=VPC(name_prefix="cluster-vpc", num_public_subnet=3, num_private_subnet=2)
```

Comment:
I'll try to create the cluster in 3 subnets / 3 AZ and then remove one so we don't have to assume the number of AZ available in the region.

Reply:
My previous approach *obviously* didn't work because subnets "should belong to the exact set of AZs [...] in which subnets were provided during cluster creation".
I finally have a working solution: get the number of AZs in the region, create N+1 subnets and update the first subnet with the last one as it is in the same AZ.